packages/components/button/stories/button.happo.js

```diff
@@ -0,0 +1,70 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '@tradeshift/elements.button';
+
+import { sizes, types } from '../src/utils';
+
+const createComponent = (componentName, content, attrs) => {
+	const element = document.createElement(`ts-${componentName}`);
+	Object.keys(attrs).forEach(attrKey => {
+		element[attrKey] = attrs[attrKey];
+	});
+	element.innerText = content;
+	element.style = 'margin-bottom: 10px;';
+	return element;
+};
+
+storiesOf('ts-button', module).add('test', () => {
+	const templates = [];
+	const label = 'Button';
+	const properties = {
+		type: types,
+		size: sizes,
+		disabled: { true: true },
+		busy: { true: true },
+		icon: { 'arrow-up': 'arrow-up' }
+	};
+	const propKeys = Object.keys(properties);
+	const componentName = 'button';
+
+	propKeys.forEach(propKey => {
+		const propTemplates = [];
+		Object.keys(properties[propKey]).forEach(key => {
+			const attrs = { [propKey]: properties[propKey][key] };
+			const element = createComponent(componentName, `${key} ${label}`, attrs);
+
+			propTemplates.push(element);
+			propKeys.forEach(otherPropKey => {
+				if (otherPropKey !== propKey) {
+					Object.keys(properties[otherPropKey]).forEach(otherKey => {
+						attrs[otherPropKey] = properties[otherPropKey][otherKey];
+						const mixedElement = createComponent(componentName, `${key} + ${otherKey} ${label}`, attrs);
+						propTemplates.push(mixedElement);
+					});
+				}
+			});
+		});
+
+		templates.push({
+			prop: propKey,
+			templates: propTemplates
+		});
+	});
+	return html`
+		${templates.map(item => {
+			return html`
+				<div class="column">
+					<h2>${item.prop}</h2>
+					${item.templates.map(propTemplate => {
+						console.log('propTemplate', propTemplate);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS2228)